setup.py

```diff
@@ -40,23 +40,30 @@
         'django==3.0.9',
         'django-admin-display',
         'django-allauth',
-        'django-composed-configuration[dev,prod]>=0.11.0',
         'django-configurations[database,email]',
         'django-extensions',
         'django-filter',
         'django-guardian',
-        'django-oauth-toolkit==1.3.2',
-        'django-s3-file-field[minio,boto3]',
+        'django-oauth-toolkit',
         'djangorestframework',
         'drf-extensions',
         'drf-yasg',
         'httpx',
         # Production-only
+        'django-composed-configuration[prod]',
+        'django-s3-file-field[boto3]',
         'django-storages[boto3]',
         'gunicorn',
-        # Development-only
-        'django-debug-toolbar',
+        # Development-only, but required
         'django-minio-storage',
```

Comment:
`django-minio-storage` is directly referenced in local project code, but I really don't think it needs to be. The current content of `storage.py` can probably be cut, especially once `django` is upgraded.

@dchiquito We should talk more about this.